lib/models/system.js

```diff
@@ -47,6 +49,50 @@ export function SystemModel () {
 
 _.extend(SystemModel.prototype, KadiraModel.prototype);
 
+function meminfo () {
+  let info = {};
+  let data = fs.readFileSync('/proc/meminfo').toString();
```

Comment:
We'll want this to be async, same with the execSync to avoid blocking the server. One option would be to run this every 2 seconds like we do for measuring cpu usage, and have buildPayload use the last value.

Reply:
got it!